### PR TITLE
Fix setup

### PR DESCRIPTION
**Pull request by nzolotova**

### CODE REVIEW

File: common/tasks/main.yml [hunk 1]
```diff
@@ -34,46 +34,39 @@
...
     - iterm2
     - sublime-text
     - macdown
     - atom
-    - dropbox
-    - google-drive
     - dash
     - disk-inventory-x
     - keepassx
-    - mattermost
```
Comment (jensfischer1515) [inline common/tasks/main.yml]: this is still used in day-to-day work!

File: hosts [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [local]
-localhost ansible_python_interpreter=/usr/local/bin/python
+localhost ansible_python_interpreter=/usr/bin/python
```
Comment (jensfischer1515) [inline hosts:2]: 👍  this is even working on my Mac OS X Sierra

File: common/tasks/main.yml (hunk 1 above)
Comment (mduesterhoeft) [reply common/tasks/main.yml]: But on Natalia's Mac it was pre installed by IT - which lets the script crash because the app is not installed via homebrew

Comment (mduesterhoeft) [reply common/tasks/main.yml]: I suppose it is now a standard app so we can assume it is always there already

Comment (nzolotova) [reply common/tasks/main.yml]: In anyway I suppose these packages are not mandatory and can be install by demand if needed.

Comment (jensfischer1515) [reply common/tasks/main.yml]: Let's try to extract mattermost into it's own task with `ignore_errors: yes`, see http://docs.ansible.com/ansible/latest/playbooks_error_handling.html

**Pull request merged**